.github/workflows/ci-build.yml

```diff
@@ -27,11 +27,24 @@ jobs:
         run: yarn type-check
       - name: Format
         run: yarn format:check
+      - name: Set env var if base is main
+        run: |
+          echo "USE_PROD_DB=true" >> $GITHUB_ENV
+          echo "NEXT_PUBLIC_IS_STAGING=false" >> $GITHUB_ENV
+        if: startsWith(github.base_ref, 'refs/heads/main')
+
+      - name: Set env var if base is staging
+        run: |
+          echo "USE_PROD_DB=false" >> $GITHUB_ENV
+          echo "NEXT_PUBLIC_IS_STAGING=true" >> $GITHUB_ENV
+        if: startsWith(github.base_ref, 'refs/heads/staging')
       - name: Test
         env:
           FIREBASE_PRIVATE_KEY: ${{ secrets.FIREBASE_PRIVATE_KEY }}
           FIREBASE_PRIVATE_KEY_ID: ${{ secrets.FIREBASE_PRIVATE_KEY_ID }}
```

Comment:
I think these need to be configured to use the dev firebase credentials when for CI tests into staging